functors_task_1/student/include/impl.h

```diff
@@ -16,16 +26,24 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
```

Comment:
_Mark "in" function params const by default, because their modification is considered as a bad practice_

Reply:
Done